welcomescreen.py

```diff
@@ -105,12 +108,4 @@ def run(self):
             if crashed:                        # Game crash or Close check
                 pygame.quit()
                 sys.exit()
-
-        # Just a window exception check condition
-        event1 = pygame.event.get()
-        if event1.type == pygame.QUIT:
-            crashed = True
-
-        if crashed:
-            pygame.quit()
-            sys.exit()
+        
```

Comment:
Removed the part where events were dequeued for the second time.

Reply:
Again.